perun-core/src/main/java/cz/metacentrum/perun/core/bl/AuditMessagesManagerBl.java

```diff
@@ -1,17 +1,14 @@
 package cz.metacentrum.perun.core.bl;
```

Comment:
I guess, you don't have to go to the Bl and Impl layer. Focus mostly on RPC a Api part of core. Just fix it if it's clearly wrong.